Tasks/AppCenterDistribute/appcenterdistribute.ts

```diff
@@ -159,8 +159,12 @@ function publishRelease(apiServer: string, releaseUrl: string, releaseNotes: str
 
     let publishBody = {
         "status": "available",
-        "distribution_group_id": distributionGroupId,
-        "release_notes": releaseNotes
+        "release_notes": releaseNotes,
+        "destinations": [
+            {
+                "id": distributionGroupId
```

Comment:
Does the request have to indicate what version of the API it's calling in some way? Or does the server handle both old and new style requests?

Or is the current version of the task broken?

Reply:
The reasonable question - both APIs are currently available - http://openapi.appcenter.ms/#/distribute/releases_update

Reply:
The version is in the in the URL `/0.1/...` however, we are working on moving the entire team to v1.0. 

Both APIs are currently available.  However, we are planning to deprecate old approach.